File: Archive/fizzbuzz/golang/fizzbuzz2.go
package main

import (
	"os"
	"strconv"
)

func main() {
	for i := 1; i < 15000001; i++ {
		if i%3 == 0 && i%5 == 0 {
			os.Stdout.Write([]byte("fizzbuzz"))
		} else if i%3 == 0 {
			os.Stdout.Write([]byte("fizz"))
		} else if i%5 == 0 {
			os.Stdout.Write([]byte("buzz"))
		} else {
			os.Stdout.Write([]byte(strconv.Itoa(i)))
		}
	}
}

File: Archive/fizzbuzz/rust/fizzbuzz2.rs
use std::io::Write;

fn main() {
    let mut stdout = std::io::stdout();
    for i in 1..=15000000 {
        match (i % 3, i % 5) {
            (0, 0) => writeln!(&mut stdout, "fizzbuzz"),
            (0, _) => writeln!(&mut stdout, "fizz"),
            (_, 0) => writeln!(&mut stdout, "buzz"),
            (_, _) => writeln!(&mut stdout, "{}", i),
        };
    }
}
